.eslint/rules/icon-has-viewBox.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @fileoverview All Paste icon svgs should have a viewBox attribute
+ * @author Glorilí Alejandro
+ */
+
+// TODO: have this only run on our icons package in its src directory
```

Comment:
This is something we want, right? or do we want this rule to apply everywhere?

Reply:
💭 Thought: I could go either way here. It looks like we do have a handful of failures in svgs outside of our icons library, so if we keep the scope global we'll want to fix those. From local testing, filtering to just the icons package doesn't have a lot of perf advantages, so might be fine to just have it work globally.

Reply:
sweet, I made it global and fixed those errors so we should be good to go!